tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/deployment/ConfigurationUtil.java

```diff
@@ -0,0 +1,9 @@
+package org.jboss.eap.qe.microprofile.tooling.server.configuration.deployment;
+
+public class ConfigurationUtil {
```

Comment:
Sorry for chiming in, but I noticed this, which I had to deal with previously, and I thought whether it'd be better to use a static file, rather than a string literal, as for example:

```
...
.addAsManifestResource("empty-beans.xml")
...
```

since it would be more easily maintained.

the Javadoc goes like this:

```
    /**
     * Adds the resource as a Manifest resource to the container, returning the container itself. <br/>
     * The resource will be placed into the Container Manifest path under the same context from which it was retrieved. <br/>
     * <br/>
     * The {@link ClassLoader} used to obtain the resource is up to the implementation.
     *
     * @param resourceName
     *            resource to add
     * @return This virtual archive
     * @throws IllegalArgumentException
     *             if resourceName is null
     * @throws IllegalArgumentException
     *             if target is null
     */
    T addAsManifestResource(String resourceName) throws IllegalArgumentException;
```    

The microprofile-jwt submodule includes even a more precise example, where a known classloader is used to retrieve and pass the resource directly, see https://github.com/jboss-eap-qe/eap-microprofile-test-suite/blob/master/microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/compatibility/JsonWebTokenApiTest.java#L50

WDYT @tommaso-borgato ?

Reply:
Tnks @fabiobrz, I moved the file content to a static file as you suggested

Reply:
Thanks for addressing my comment, @tommaso-borgato !

BTW, just thinking about it, is there any reason for not doing something like the following?

```
.addAsManifestResource(ConfigurationUtil.BEANS_XML_FILE_LOCATION, "beans.xml")
```

or even:
 
```
.addAsManifestResource(ConfigurationUtil.class.getClassLoader().getResource("cdi/beans.xml"), "beans.xml")
```

and remove the whole `readFromInputStream()` method?

Or am I missing something?

Reply:
tnks @fabiobrz, that leads to more compact code and is definitely an improvement!

Reply:
`readFromInputStream` doesn't seems to be used, ConfigurationUtil.class.getClassLoader().getResource("cdi/beans.xml") is stored in ConfigurationUtil for easy usage in a lot of cases. So this thread seems to be ready to resolve, @fabiobrz WDYT?

Reply:
+1, resolving, thanks!